metapool/sample_sheet.py

```diff
@@ -1366,13 +1364,32 @@ class TellseqMetagSampleSheetv10(KLTellSeqSampleSheet):
     _HEADER[_SHEET_VERSION_KEY] = '10'
     _HEADER[_ASSAY_KEY] = _METAGENOMIC
 
+    @property
+    def CARRIED_PREP_COLUMNS(self):
+        return [x for x in _BASE_CARRIED_PREP_COLUMNS if x not in
+                {'i7_index_id', 'index', 'index2', 'i5_index_id'}]
+
+    @property
+    def GENERATED_PREP_COLUMNS(self):
+        return list(_BASE_GENERATED_PREP_COLUMNS)
+
 
 class TellseqAbsquantMetagSampleSheetv10(AbsQuantMixin, KLTellSeqSampleSheet):
     _HEADER = KLSampleSheet._HEADER.copy()
     _HEADER[_SHEET_TYPE_KEY] = TELLSEQ_ABSQUANT_SHEET_TYPE
     _HEADER[_SHEET_VERSION_KEY] = '10'
     _HEADER[_ASSAY_KEY] = _METAGENOMIC
 
+    @property
+    def CARRIED_PREP_COLUMNS(self):
+        return [x for x in _BASE_CARRIED_PREP_COLUMNS if x not in
+                {'i7_index_id', 'index', 'index2', 'i5_index_id'}] + \
+                list(AbsQuantMixin._ABSQUANT_SPECIFIC_COLUMNS)
+
```

Comment:
I understand the necessity of adding these properties in whatever way works for now to get us ready to run.  Could you add an issue to clean these up (if not already created) and drop a comment on both of these referencing it?  I am thinking something like: 

`# If refactoring, see issue #XXX`

In theory I guess I should check the whole list of issues every time I work on the code, but I don't ;) so issues that aren't breadcrumbed at the piece of code that needs the work tend to get forgotten by me.